.editorconfig

```diff
@@ -0,0 +1,26 @@
+root = true
+
+[*]
+trim_trailing_whitespace = true
+end_of_line = lf
+
+[*.py]
+indent_style = space
+indent_size = 4
+charset = utf-8
+insert_final_newline = true
+max_line_length = 120
+
+
+[*.{js,css}]
```

Comment:
The style guides we use combine HTML and CSS into one style guide, and have a separate one for JS, so I suggest making this block just for JS and putting CSS together with HTML below.